<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visual Minimum Spanning Tree</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="separator"/>
    <addaction name="actionSelectMode"/>
    <addaction name="actionMoveMode"/>
    <addaction name="actionPointMode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="separator"/>
   <addaction name="actionSelectMode"/>
   <addaction name="actionMoveMode"/>
   <addaction name="actionPointMode"/>
   <addaction name="separator"/>
   <addaction name="actionFindMST"/>
   <addaction name="separator"/>
   <addaction name="actionNewPoint"/>
   <addaction name="actionRemovePoint"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/document-open.png</normaloff>:/new/icons/res/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/document-new.png</normaloff>:/new/icons/res/document-new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/document-save.png</normaloff>:/new/icons/res/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/tools_generaltools_zoom_in.png</normaloff>:/new/icons/res/tools_generaltools_zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/tools_generaltools_zoom_out.png</normaloff>:/new/icons/res/tools_generaltools_zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionSelectMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/tools_movement_move.png</normaloff>:/new/icons/res/tools_movement_move.png</iconset>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
   <property name="toolTip">
    <string>Select</string>
   </property>
  </action>
  <action name="actionMoveMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/arrow_cursor_grab.png</normaloff>:/new/icons/res/arrow_cursor_grab.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="toolTip">
    <string>Move</string>
   </property>
  </action>
  <action name="actionPointMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/tools_point_new_point.png</normaloff>:/new/icons/res/tools_point_new_point.png</iconset>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
   <property name="toolTip">
    <string>Draw</string>
   </property>
  </action>
  <action name="actionFindMST">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/tools_spreadsheet_sum.png</normaloff>:/new/icons/res/tools_spreadsheet_sum.png</iconset>
   </property>
   <property name="text">
    <string>FindMST</string>
   </property>
   <property name="toolTip">
    <string>Find MST</string>
   </property>
  </action>
  <action name="actionNewPoint">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/properties_object_list_add.png</normaloff>:/new/icons/res/properties_object_list_add.png</iconset>
   </property>
   <property name="text">
    <string>NewPoint</string>
   </property>
   <property name="toolTip">
    <string>New Point</string>
   </property>
  </action>
  <action name="actionRemovePoint">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/icons/res/Image-button_cancel.png</normaloff>:/new/icons/res/Image-button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>RemovePoint</string>
   </property>
   <property name="toolTip">
    <string>Remove Point</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
